lib/models/BucketInfo.ts

```diff
@@ -37,6 +37,41 @@ export type VersioningConfiguration = {
     MfaDelete: any;
 };
 
+export type VeeamCapacity = {
+    SystemInfo?: {
+        ProtocolVersion: string,
+        ModelName: string,
+        ProtocolCapabilities: {
+            CapacityInfo: boolean,
+            UploadSessions: boolean,
+            IAMSTS?: boolean,
+        },
+        APIEndpoints?: {
+            IAMEndpoint: string,
+            STSEndpoint: string,
+        },
+        SystemRecommendations?: {
+            S3ConcurrentTaskLimit: number,
+            S3MultiObjectDelete: number,
+            StorageCurrentTasksLimit: number,
+            KbBlockSize: number,
+        }
+        LastModified?: string,
+    },
+    CapacityInfo?: {
+        Capacity: number,
+        Available: number,
+        Used: number,
+        LastModified?: string,
+    },
+};
+
+// Capacity contains all specifics from external products supported by
+// our S3 implementation, at bucket level
+export type Capacity = {
```

Comment:
this type should be `Capabilies` : it may contain 'capacity' for veam, but other info if we do other integration...

Reply:
Done: https://github.com/scality/Arsenal/pull/2059/files#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R71